src/pattern-library/components/patterns/feedback/ModalPage.tsx

```diff
@@ -388,6 +424,40 @@ export default function ModalPage() {
               </Modal_>
             </Library.Demo>
           </Library.Example>
+
+          <Library.Example title="scrollable (default true)">
+            <p>
+              By default, Modals will scroll overflowing content. In some cases,
+              it is desireable to scroll only a portion of the content, or to
+              disable scrolling entirely.
+            </p>
+            <p>
+              This example demonstrates setting the boolean{' '}
+              <code>scrollable</code> prop to <code>false</code> and using a{' '}
+              <code>Scroll</code> component within Modal content to scroll only
+              a portion of the content.
+            </p>
+            <Library.Demo title="Manually controlling scrolling" withSource>
+              <Modal_
+                buttons={<ModalButtons />}
+                classes="h-[25rem]"
+                onClose={() => {}}
+                title="Modal with scrollable disabled"
+                scrollable={false}
+              >
+                <p>
+                  Fake {'>'} breadcrumbs {'>'} example
+                </p>
+                <Scroll>
+                  <DataTable
+                    rows={nabokovRows}
+                    columns={nabokovColumns}
+                    title="Nabokov novels"
+                  />
+                </Scroll>
+              </Modal_>
```

Comment:
_This_ is the core use case I'm solving for here: A Modal with some non-scrolling content (like breadcrumbs at the top of a list of files) and a Table that _should_ scroll.